### ISSUE TITLE
Game Library bounds in landscape not within safe margins for iPhone X

### ISSUE THREAD

**Issue by sevdestruct** [jasarien/Provenance#656]
Navigation controller looks to be within safe margins, but the game Library cells are falling outside of them, causing cover art to get clipped under the notch.

![61e52052-b20a-4793-b11c-a9c38cc3ed01](https://user-images.githubusercontent.com/3118097/34966790-49028f10-fa13-11e7-8d52-23019e5dc9ff.png)

with a wider screen on the X, might not hurt to have less columns in landscape orientation for it anyhow, as it jumps to 7 which is quite a lot to scan visually, considering.

**Comment by braindx** [comment]
+1 to this. If I had any idea what I was doing with iOS UI I would fix this.

**Comment by sevdestruct** [comment]
hey @braindx , I took a look into it setting it straight, when i posted this but coudn't sort it - though I didn't but nudge some constraints at a glance.. so I can't say i really tried much on this one.

Considering the recent talk of UI needs, I am thinking instead of revisiting this and additonal cleanups to the existing UI that I submitted that I might start sorting out new UI design from the ground up,. I have had a lot of ideas since I joined up and since using the app -- some of which I've posted in various issue threads here -- my background/profession is design actually (code is more of a nascent hobby WIP skill). 

That said, after I complete work on dark mode, and when I have some free time I am going to jump into Sketch and start mocking up screens to see how I might solve some of the UI/UX needs that have arisen, especially with the expanding cores, and other improvements I'd like to pursue.

That reminds me of something I was going to ask Joe..
